hr_timesheet_improvement/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Timesheet improvements
+======================
+
+Modifies timesheet behavior:
+ * Ensure a DESC order on timesheet lines
+ * Set default date for manually entering attendance to max attendance date
+ * Redefine constraint on timesheets to check alternation of 'sign in' and
+   'sign out' only on current timesheet instead of doing it on all timesheets
+   of the employee
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/hr-timesheet/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/hr-timesheet/issues/new?body=module:%20hr_timesheet_improvement%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Vincent Renaville <vincent.renaville@camptocamp.com>
+* Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
+* Yannick Vaucher <yannick.vaucher@camptocamp.com>
+* Pedro M. Baeza <pedro.baeza@gmail.com>
```

Comment:
Indeed my work was very little in this module, so technically I'm a contributor, but doesn't mind if I appear or not. It's up to you.